### ISSUE TITLE
why pretrainedModel "dcp_v2.t7" include more parameters?

### ISSUE THREAD

**Issue by qiaozhijian** [WangYueFt/dcp#13]
When I set "strict" "True" in load_state_dict, I find errors "Unexpected key(s) in state_dict:...". Because in "pointer.model.encoder.layers.0.sublayer.0.norm", there are redundant parameters.
for example:
44 pointer.model.encoder.layers.0.sublayer.0.norm.norm.weight
45 pointer.model.encoder.layers.0.sublayer.0.norm.norm.bias
46 pointer.model.encoder.layers.0.sublayer.0.norm.norm.running_mean
47 pointer.model.encoder.layers.0.sublayer.0.norm.norm.running_var
48 pointer.model.encoder.layers.0.sublayer.0.norm.norm.num_batches_tracked

**Comment by WangYueFt** [comment]
I think that's because I had extra normalize layer (which was not used at all) when I saved  the checkpoint. So no need to worry about that.
